tests/unit/common/components/input.spec.js

```diff
@@ -0,0 +1,145 @@
+import { mount } from "@vue/test-utils"
+import Input from "@/common/components/Input"
+// import File from "@/common/components/File"
+// import Button from "@/common/components/Button"
+import Vue from "vue"
+import Vuetify from  "vuetify"
+
+Vue.use(Vuetify)
+
+describe("Input component", () => {
+
+  let wrapper
+
+  beforeEach(() => {
+    const nodeCrypto = require("crypto")
+
+    global.crypto = {
+      getRandomValues: (buffer) => { return nodeCrypto.randomFillSync(buffer) }
+    }
+  })
+
+  it("Should render", () => {
+    wrapper = mount(Input)
+
+    expect(wrapper.exists()).toBe(true)
+  })
+
+  it("Should render with default autocomplete off", () => {
+    wrapper = mount(Input)
+
+    const inputComponent = wrapper.find(".ui-debio-input__input")
+    expect(inputComponent.attributes("autocomplete")).toBe("off")
+  })
+
+  it("Should render with automplete on", () => {
+    wrapper = mount(Input, {
+      propsData: { autocomplete: "on " }
+    })
+
+    const inputComponent = wrapper.find(".ui-debio-input__input")
+    expect(inputComponent.attributes("autocomplete")).toBe("on ")
+  })
+
+  it("Should render with default spellcheck off", () => {
+    wrapper = mount(Input)
+
+    const inputComponent = wrapper.find(".ui-debio-input__input")
+    expect(inputComponent.attributes("spellcheck")).toBe("false")
+  })
+
+  it("Should render with spellcheck", () => {
+    wrapper = mount(Input, {
+      propsData: { spellcheck: "on " }
+    })
+
+    const inputComponent = wrapper.find(".ui-debio-input__input")
+    expect(inputComponent.attributes("spellcheck")).toBe("true")
+  })
+
+  it("Should render with default empty label", () => {
+    wrapper = mount(Input)
+
+    const inputComponent = wrapper.find(".ui-debio-input__label")
+    expect(inputComponent.exists()).toBe(false)
+  })
+
+  it("Should render with custom label", () => {
+    wrapper = mount(Input, {
+      propsData: { label: "input" }
+    })
+
+    const inputComponent = wrapper.find(".ui-debio-input__label")
+    expect(inputComponent.attributes("aria-label")).toBe("input")
+  })
+
+  it("Should render default width", () => {
+    wrapper = mount(Input)
+
+    const inputComponent = wrapper.find(".ui-debio-input")
+    expect(inputComponent.attributes("style")).toBe("width: 200px;")
+  })
+
+  it("Should render default width", () => {
```

Comment:
this supposed to be custom width, right?

Reply:
yess, i think this should be custom width. iam sorry my bad